plugin-server/src/capabilities.ts

```diff
@@ -9,13 +9,16 @@ export function getPluginServerCapabilities(config: PluginsServerConfig): Plugin
         case null:
             return {
                 ingestion: true,
+                ingestionOverflow: true,
                 pluginScheduledTasks: true,
                 processPluginJobs: true,
                 processAsyncHandlers: true,
                 ...sharedCapabilities,
             }
         case 'ingestion':
             return { ingestion: true, ...sharedCapabilities }
+        case 'ingestion-overflow':
+            return { ingestionOverflow: true, ...sharedCapabilities }
```

Comment:
i.e. like this to start with, before removing ingestionOverflow: true once we're merged the deployment change.

```suggestion
        case 'ingestion':
            return { ingestion: true, ingestionOverflow: true, ...sharedCapabilities }
        case 'ingestion-overflow':
            return { ingestionOverflow: true, ...sharedCapabilities }
```